format/base.go

```diff
@@ -11,17 +11,17 @@ import (
 )
 
 // bold converts the provided text to bold
```

Comment:
can you update the names in the documentation to match the new exported function names?

```suggestion
// Bold converts the provided text to bold
```